Admin/BaseFieldDescription.php

```diff
@@ -422,10 +422,14 @@ public function getMappingType()
      * @param string $property
      *
      * @return string
+     *
+     * @deprecated Deprecated since version 3.1. Use \Doctrine\Common\Inflector\Inflector::camelize() instead.
      */
     public static function camelize($property)
     {
-        return Container::camelize($property);
+        @trigger_error('The method '.__METHOD__.' is deprecated since 3.1 and will be removed in 4.0. Use \Doctrine\Common\Inflector\Inflector::camelize() instead.', E_USER_DEPRECATED);
```

Comment:
```php
@trigger_error(
    sprintf(
        'The %s method is deprecated since 3.1 and will be removed in 4.0. '.
        'Use \Doctrine\Common\Inflector\Inflector::camelize() instead.',
        __METHOD__
    )
    E_USER_DEPRECATED
);
```
would look nicer